packages/dev/gui/src/3D/controls/touchHolographicButton.ts

```diff
@@ -391,6 +399,14 @@ export class TouchHolographicButton extends TouchButton3D {
         this.collisionMesh = collisionMesh;
         this.collidableFrontDirection = this._backPlate.forward.negate(); // Mesh is facing the wrong way
 
+        this._pointerObserver = scene.onPointerObservable.add((pointerInfo) => {

```

Comment:
This must be removed when the component is disposed. It's also the reason the build failed.

Reply:
Also should be removed if this function is called again (or if already defined) - stay defensive

Reply:
Ah, yes! I must've missed it. I'll resolve it, thanks! :)